azurerm/resource_arm_postgresql_database_test.go

```diff
@@ -0,0 +1,124 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMPostgreSQLDatabase_basic(t *testing.T) {
+	resourceName := "azurerm_postgresql_database.test"
+	ri := acctest.RandInt()
+	config := testAccAzureRMPostgreSQLDatabase_basic(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMPostgreSQLDatabaseDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMPostgreSQLDatabaseExists(resourceName),
```

Comment:
Could we add more checks here? At least state checks via `resource.TestCheckResourceAttr*` would be nice.

Reply:
Done.